docs/essentials/controls.md

```diff
@@ -45,9 +87,57 @@ By default, Storybook will choose a control for each arg based on its initial va
 
 </IfRenderer>
 
+<IfRenderer renderer='ember'>
+
+By default, Storybook will try to infer the required argTypes and associated controls for your stories based on the metadata provided by the [`@storybook/ember-cli-storybook`](https://github.com/storybookjs/ember-cli-storybook) addon. You'll need to take some additional steps to set it up properly.
+
+Update your `ember-cli-build.js` configuration file to include the addon.
```

Comment:
Calling this package an addon seems confusing? "Adapter" is the term used by the README.

```suggestion
By default, Storybook will try to infer the required argTypes and associated controls for your stories based on the metadata provided by the [`@storybook/ember-cli-storybook`](https://github.com/storybookjs/ember-cli-storybook) adapter. You'll need to take some additional steps to set it up properly.

Update your `ember-cli-build.js` configuration file to include the adapter.
```